pages/pet-detail-page/[id].js

```diff
@@ -345,6 +345,11 @@ export default function PetDetailPage({
             <MoneyColored cost={200} money={userStats.money} /> <MoneyImage />
           </StyledButton>
         )}
+        <StyledButton
+          onClick={() => router.push(`/${id}/minigames/obstacle-jumper`)}
+        >
+          Obstacle Jumper
+        </StyledButton>
```

Comment:
Is there a reason why this is not a link?

Reply:
will be a popup in future, that show all minigames